src/executor/query.rs

```diff
@@ -115,6 +115,27 @@ impl QueryResult {
     {
         Ok(T::try_get_many_by_index(self)?)
     }
+
+    /// Gets all column names in the order they were written.
+    #[cfg(feature = "sqlx-dep")]
```

Comment:
I think if we move this line to directly above `use sqlx::Column`, this can still compile?